integration/airflow/requirements.txt

```diff
@@ -213,12 +207,12 @@ idna==3.6
     #   httpx
     #   requests
     #   yarl
-importlib-metadata==6.11.0
+importlib-metadata==7.0.0
```

Comment:
multiple dependencies are being upgraded to new major versions. how do we know that this won't break things? can we skip dependabot on integrations?